marpa/lib/Guacamole.pm

```diff
@@ -804,6 +804,11 @@ OpKeywordSocketExpr           ::= OpKeywordSocket Expression OpComma Expression
 
 OpKeywordSocketpairExpr       ::= OpKeywordSocketpair Expression OpComma Expression OpComma Expression OpComma Expression OpComma Expression
 
+OpKeywordSortExpr             ::= OpKeywordSort Block Expression
+                                | OpKeywordSort NonBraceExpression Expression
+                                | OpKeywordSort VarScalar Expression
+                                | OpKeywordSort Expression
```

Comment:
This should be `NonBraceExpression` because it follows a token that can also be followed by a `Block`.

Reply:
Fixed.